cmd/platform/channel.go

```diff
@@ -205,7 +219,7 @@ func addUserToChannel(channel *model.Channel, user *model.User, userArg string)
 	}
 }
 
-func deleteChannelsCmdF(cmd *cobra.Command, args []string) error {
+func archiveChannelsCmdF(cmd *cobra.Command, args []string) error {
```

Comment:
@JeffSchering the wording is wrong in the archive channels I will submit a PR to change that.

for the restore feature is not available to other editions I guess is intentional, but @coreyhulen can confirm that.